src/main/java/uk/gov/hmcts/reform/pbis/model/PrivateBetaRegistration.java

```diff
@@ -1,20 +1,31 @@
 package uk.gov.hmcts.reform.pbis.model;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 public final class PrivateBetaRegistration {
 
+    @JsonProperty
     public final String referenceId;
+
+    @JsonProperty
     public final String service;
+
+    @JsonProperty
     public final String emailAddress;
+
+    @JsonProperty
     public final String firstName;
+
+    @JsonProperty
```

Comment:
are those needed?

Reply:
Probably the ones in the constructor are enough. I'll remove them.

Reply:
Done